### ISSUE TITLE
 BUG: Persistent Border on "Are you a seller" in Navbar Across All Pages

### ISSUE THREAD

**Issue by Amishachoudhary22** [Anushkabh/krishiconnect#661]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

There is an issue with the navbar where the "Are you a seller" link has a persistent border regardless of the page the user is on. This unintended styling issue affects the visual consistency of the navigation bar across the site.

Steps to Reproduce:

Navigate to any page on the website (e.g., home, about, contact).
Observe the navbar.
Note that the "Are you a seller" link has a border.
Expected Behavior:
The "Are you a seller" link in the navbar should not have a border unless it is specifically styled to do so (e.g., when active or focused).

Actual Behavior:
The "Are you a seller" link in the navbar consistently displays a border, regardless of its state or the page being viewed.
I want to fix it such that the border appears on whichever page we navigate.
<img width="1419" alt="Screenshot 2024-06-23 at 12 58 00 PM" src="https://github.com/Anushkabh/krishiconnect/assets/103454929/0ff63f88-cb4a-4f18-adcf-e3da57bad43a">

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by himanshu-03** [comment]
hey @Amishachoudhary22 !

The border is because it serves as a login and register page for the seller. if not  border we could add some hover on or change the color of the box so that one can differentiate between a normal nav link and a login link.

Do let me know your thoughts on this!

**Comment by Amishachoudhary22** [comment]
I agree that the border provides a clear indication of the login or register page. However, I also see the value in ensuring that the visual cue changes according to the page the user is on, which can provide additional clarity.

Yeah we could possibly use hover effects or color changes to distinguish these links from regular navigation links. This way, users can still easily identify the login and register options without the potential confusion of thinking they are already on those pages when they are not.

By implementing dynamic visual indicators that adjust based on the user's location within the site, we can enhance the overall user experience and improve navigation clarity. Thank you for your valuable feedback, and I look forward to exploring these options further.

**Comment by himanshu-03** [comment]
> Hey @himanshu-03 I agree that the border provides a clear indication of the login or register page. However, I also see the value in ensuring that the visual cue changes according to the page the user is on, which can provide additional clarity.
> 
> Yeah we could possibly use hover effects or color changes to distinguish these links from regular navigation links. This way, users can still easily identify the login and register options without the potential confusion of thinking they are already on those pages when they are not.
> 
> By implementing dynamic visual indicators that adjust based on the user's location within the site, we can enhance the overall user experience and improve navigation clarity. Thank you for your valuable feedback, and I look forward to exploring these options further.

Okay @Amishachoudhary22 !

Currently we are updating the header on our website once it is done you can start working on the issue I will let you know when it is ready.

happy coding ✨

**Comment by Amishachoudhary22** [comment]
Alright @himanshu-03 I will look forward to it.
